contents/node_operator.rst

```diff
@@ -412,6 +412,101 @@ If the external node is running on IP 12.34.56.78 port 8545, the command would b
 
 You can also use ``https``. But keep in mind that Swarm *does not validate the certificate*.
 
+Connect to the SWAP-enabled testnet
+===================================
+The Swarm project now runs a SWAP-enabled (incentivized) testnet. It uses the same binary as the normal network node, but has incentivization switched on. 
+
+.. important::
+
+  The testnet is highly experimental and may be updated continuously until we release a mainnet version. All tokens used on this testnet are fictitious. Do NOT use this testnet with real value tokens. Funds can be lost.
+
+ 
+The first public incentives-enabled Swarm network runs on the ropsten network. 
+It runs with network id **5**
+
+Prerequisites
+-----------------
+* Create an account on **ropsten**
+* Fund your newly created account with test coins
+
+Network setup
+-------------
+We provide on the public swap-enabled network currently 30 nodes in a kubernetes cluster. 
```

Comment:
Pretty sure "currently" should go either to the beginning or the end of the sentence.